setup.py

```diff
@@ -18,6 +18,7 @@ def read(filename):
         filename)
     return open(full_filename, mode='r', encoding='utf-8').read()
 
+
```

Comment:
It seems that flake8 behaves differently when I run it locally, this commit can probably be ignored.